api/server/server.go

```diff
@@ -1213,6 +1213,7 @@ func (s *Server) postBuild(version version.Version, w http.ResponseWriter, r *ht
 	}
 
 	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Server", "Docker/"+dockerversion.VERSION+" ("+runtime.GOOS+")")
```

Comment:
There is one thing - this is getting set for /build only. I can't see an obvious better place it should be to cover all API calls